state-sync/storage-service/server/src/lib.rs

```diff
@@ -105,7 +120,7 @@ impl<T: StorageReaderInterface> Handler<T> {
     }
 
     pub fn call(&self, request: StorageServiceRequest) -> Result<StorageServiceResponse> {
-        let response = match request {
+        let response = match request.clone() {
             StorageServiceRequest::GetAccountStatesChunkWithProof(request) => {
                 self.get_account_states_chunk_with_proof(request)
             }
```

Comment:
You could also pass in the reference to the request.  These requests can get quite large right?

Reply:
Generally, all of these requests are small (simple structs with 2-3 numbers), but references are much easier 😄 Done!